src/boot/ksdk1.1.0/devMAG3110.h

```diff
@@ -35,7 +35,9 @@
 	POSSIBILITY OF SUCH DAMAGE.
 */
 
-
+#ifndef warp_devMAG3110_ENABLE
```

Comment:
Please see the coding conventions at https://github.com/physical-computation/Warp-firmware/blob/master/doc/README-CodingConventions.md